target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8065-r7800.dts

```diff
@@ -45,6 +45,13 @@
 			pinctrl-0 = <&mdio0_pins &rgmii2_pins>;
 			pinctrl-names = "default";
 
```

Comment:
@dissent1 why You move this pinctrl-0 from mdio?
https://github.com/lede-project/source/commit/eb7307cb943bfe4f10fa2f77abbe71dfcbbb3601#diff-8a61e3ed51bbb9e79031d387b2066417